src/twisted/runner/procmon.py

```diff
@@ -420,7 +420,7 @@ def __repr__(self):
 
             if uidgid:
                 uidgid = '(' + uidgid + ')'
-            l.append('%r%s: %r' % (name, uidgid, proc.args))
```

Comment:
Not a big deal, but consider using format strings

Reply:
I'd rather do that when we drop Python 3.5 support, so we can go straight to using f-strings.